kuma/wiki/search.py

```diff
@@ -153,15 +161,6 @@ def get_analysis(cls):
                         'snowball',
                     ],
                 },
-                'case_sensitive': {
```

Comment:
Any reson to remove them?

Reply:
These we useful with ES 2.x, which had a generic ``String`` datatype for full text search and keyword (non-analyzed) search. They aren't required with the ``Keyword`` datatype, which more clearly states the purpose of these fields. ``case_sensitive`` is the default for ``Keyword`` search, and we can get the equivalent of ``case_insensitive_keyword`` by lowercasing keywords on indexing and querying.

Reply:
@jwhitlock storing lowercase data and lowercase tokenizer is not same.
If you use lowercase tokenizer, and you index `"This is test"` and search with `"test"`, you will still get `"This is test"`, not `"this is test"`. If you lowercase them while indexing and searching, you will always get `"this is text"`, which breaks the search result and others.

Reply:
I'm not convinced there is a problem yet. I did a lot of manual testing of the Kumascript macro dashboard, and this new mapping worked. I might be missing something, and will re-read your comments after sleeping on it. If you can think of a good example that would work on production but breaks in this branch, it would help me understand better and could be a good candidate for a test.